ios/brave-ios/Sources/BraveWallet/Crypto/Stores/SwapTokenStore.swift

```diff
@@ -367,7 +367,8 @@ public class SwapTokenStore: ObservableObject, WalletObserverStore {
       toToken: buyAddress,
       toAmount: buyAmountInWei,
       slippagePercentage: "\(slippagePercentage)",
-      routePriority: .recommended
+      routePriority: .recommended,
+      provider: .auto
```

Comment:
@onyb maybe better we check token coin and force `.zeroX` for Ethereum / `.jupiter` for Solana until we have [updated copy](https://github.com/brave/brave-core/blob/9dd6442173b64d26804ebe8c26ef0f4d177a1bec/ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/SwapCryptoView.swift#L247) / decision for disclaimers. Unless this PR is waiting for that decision?

```suggestion
      // TODO(stephenheaps): Enable Lifi support https://github.com/brave/brave-browser/issues/36436
      provider: sellToken.coin == .sol ? .jupiter : .zeroEx
```